lighthouse-cli/chrome-launcher.ts

```diff
@@ -136,6 +136,11 @@ class ChromeLauncher {
 
   spawn(execPath: string): Promise<any[]> {
     return new Promise((resolve, reject) => {
+      if (this.chrome) {
+        log.verbose('ChromeLauncher', `Chrome already running with pid ${this.chrome.pid}.`);
```

Comment:
maybe use `log.warn`? Seems like an unusual situation that in most cases you'd want to know your use of launcher is trying to launch chrome twice